pyproject.toml

```diff
@@ -65,7 +65,7 @@ dependencies = [
     "nwa-stdlib~=1.4.8",
     "oauth2-lib~=1.3.1",
     "tabulate==0.9.0",
-    "strawberry-graphql==0.171.1"
+    "strawberry-graphql@git+https://github.com/tjeerddie/strawberry#egg=pydantic-convert-specialized-list-class"
```

Comment:
using my forked version until it gets merged

https://github.com/strawberry-graphql/strawberry/pull/2909